src/Makefile

```diff
@@ -785,6 +798,19 @@ $(SELINUX_POLICY): $(SELINUX_MODULE)
 $(SELINUX_MODULE): $(SELINUX_ENFORCEMENT)
 	checkmodule -M -m -o $@ $?
 
+ifeq (${TARGET}, agent)
+strip: ${BUILD_AGENT} ${BUILD_CMAKE_PROJECTS}
+else
```

Comment:
Add ifeq with TARGET to avoid winagent and server overlap

Reply:
Nevermind. ifneq with winagent will be a propper approach due other targets (hybrid, local)